OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/InAppMessagePresenter.java

```diff
@@ -83,6 +92,15 @@ public void appEnteredBackground() {
         // noop
     }
 
+    synchronized void setDisplayMode(@NonNull OptimoveConfig.InAppDisplay mode) {
+        boolean resumed = display != mode && mode != OptimoveConfig.InAppDisplay.PAUSED;
+        display = mode;
+
+        if (resumed) {
+            presentMessageToClient();
```

Comment:
Should setting back to AUTOMATIC show the messages? if there was an attempt to show when PAUSED, the view will be null and setting back to AUTOMATIC won't show

Reply:
Yes, that's the idea that messages are shown upon setting back to AUTO from PAUSED.

You're right the view may be null at this point (we have maybe never shown anything yet), but currently view management is handled just in time by presentMessageToClient() (https://github.com/optimove-tech/Optimove-SDK-Android/blob/4a549e96b85d5bbe5e6422600b5b9a100d098c6f/OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/InAppMessagePresenter.java#L128-L136) so it should be good.